rdrf/rdrf/static/js/show_password.js

```diff
@@ -0,0 +1,9 @@
+function initToggleShowPassword($passwordFields, $passwordToggle) {
```

Comment:
Can you add a `setTimeout` that hides the password again after 30sec/1min, in case they walk away from the screen?

As discussed, it won't prevent 'inspect-element' style attacks, but should prevent someone peering over a user's shoulder.

Do you think it's worth clearing the password after some longer time as well?

Reply:
I think for the purposes of toggling a password, 60 seconds should be adequate, particularly since the user can re-toggle the visibility again.

It may not strictly achieve the WCAG for timed content though. https://www.w3.org/TR/WCAG21/#timing-adjustable

Reply:
It might be a little more complicated, but the extend option could work:

> Extend: The user is warned before time expires and given at least 20 seconds to extend the time limit with a simple action (for example, "press the space bar"), and the user is allowed to extend the time limit at least ten times

Something like a little tooltip that pops up.

Also, typing in the password fields should probably reset the time limit

Reply:
As discussed, implemented a message that displays when the user turns on "Show password(s)" to allow the automatic hiding of passwords to be turned off.

![image](https://user-images.githubusercontent.com/60163872/135569156-5c1e5d00-9b84-41e0-869a-420a0359330b.png)